content/en/blog/2023/ambient-ebpf-redirection/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Leverage eBPF for traffic redirection in Istio ambient mode"
+description: An alternative approach to redirect application pod traffic to ztunnel in Istio ambient mode.
+publishdate: 2023-03-15
+attribution: "Iris Ding (Intel), Chun Li (Intel)"
+keywords: [istio,ambient,ztunnel,eBPF]
+---
+
+The istio-cni component running on each Kubernetes worker node is responsible for redirecting application pod traffics to ztunnel on that node. By default it relies on iptables and
+Geneve tunnel to achieve this redirection.  Now a new approach which is based on eBPF is also available in Istio ambient mode for this purpose.

```

Comment:
What is "Geneve tunnel" and why is that significant? Add a link?

Reply:
sure, will add a link  for Geneve tunnel.